api/proto/config/appconfig.proto

```diff
@@ -63,5 +82,8 @@ message AppInstanceConfig {
   CipherBlock cipherData = 13;
```

Comment:
Can this be changed without restart?

Reply:
It changes always while generating device config by controller because the initial value used for encryption changes every time.

Reply:
So there is a difference @zed-rishabh between that refresh due to crypto and an actual content change. For a content change to user data to take effect the app instance needs to be restarted. (And if it is cloud-init as opposed to container environment, depending on the domU cloud-init code might even need a purge to "reinstall" with new cloud-init user-data.)